cmd/root.go

```diff
@@ -201,10 +203,19 @@ func printCheckStart(enabledChecks checker.CheckNameToFnMap) {
 	}
 }
 
-func printProbeResults(enabledProbes []string) {
+func printProbeResults(enabledProbes []string, findings []finding.Finding) {
 	for _, probeName := range enabledProbes {
 		fmt.Fprintf(os.Stderr, "Finished probe %s\n", probeName)
 	}
+	for _, result := range findings {
+		if result.Remediation != nil {
+			// Remediation is indicative of a bad result.
+			// This output is a light alternative to the details provided by --format=probe
+			fmt.Fprintf(os.Stderr, "[%s] Remediation required: %s\n", result.Probe, result.Remediation.Text)
+		} else {
+			fmt.Fprintf(os.Stderr, "[%s] Passed: %s\n", result.Probe, result.Message)
```

Comment:
How important are these exact phrases to your use case? Just informational? 
I ask because I may want this cleaned up in the future to be part of [AsString](https://github.com/ossf/scorecard/blob/6c2be58d3899dc6b4d4c93891925d619c9afbdf1/pkg/scorecard/scorecard_result.go#L182). I'm not sure what it would look like, but being able to send the string to `os.Stdout` or the specified `--output` file would be good long-term.